Error messages during build when downloaded as .zip

The git commands in the makefile don't work then.
I'm getting:

```
fatal: Not a git repository (or any parent up to mount point /home)
Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).
Not a git repository
To compare two paths outside a working tree:
usage: git diff [--no-index] <path> <path>
fatal: Not a git repository (or any parent up to mount point /home)
Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).
fatal: Not a git repository (or any parent up to mount point /home)
Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).
Not a git repository
To compare two paths outside a working tree:
usage: git diff [--no-index] <path> <path>
fatal: Not a git repository (or any parent up to mount point /home)
Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).
echo "const char *version_str = \"arachne-pnr 0.1+0+0 (git sha1 , g++ `g++ --version | tr ' ()' '\n' | grep '^[0-9]' | head -n1` -O0 -fno-inline)\";" > src/version_02097.cc
```

It does not stop there, but continues.
It finally fails with:

```
g++ -Isrc -std=c++11 -MD -O0 -fno-inline -g -Wall -Wshadow -Wsign-compare -Werror  -o bin/arachne-pnr src/arachne-pnr.o src/netlist.o src/blif.o src/pack.o src/place.o src/util.o src/io.o src/route.o src/chipdb.o src/location.o src/configuration.o src/line_parser.o src/pcf.o src/global.o src/constant.o src/designstate.o src/version_02097.o -lm
make: *** No rule to make target '/usr/share/icebox/chipdb-1k.txt', needed by 'share/arachne-pnr/chipdb-1k.bin'.  Stop.
```

Now I'm not sure... will git clone fix it or am I missing some textfiles?

--- a/Makefile
+++ b/Makefile
@@ -19,8 +19,8 @@ ICEBOX = /usr/local/share/icebox
 .PHONY: all
 all: bin/arachne-pnr share/arachne-pnr/chipdb-1k.bin share/arachne-pnr/chipdb-8k.bin
 
-VER = 0.1+$(shell echo `git log --oneline | wc -l`)+$(shell echo `git diff --name-only HEAD | wc -l`)
-GIT_REV = $(shell git rev-parse --verify --short HEAD)
+VER = 0.1+$(shell test -e .git && echo `git log --oneline | wc -l`+`git diff --name-only HEAD | wc -l`)
+GIT_REV = $(shell git rev-parse --verify --short HEAD 2>/dev/null || echo UNKNOWN)
 
 VER_HASH = $(shell echo "$(VER) $(GIT_REV)" | sum | cut -d ' ' -f -1)
 

